### PR TITLE
Change the comment so it matches the command it describes

### PR DESCRIPTION
**Pull request by ekeih**
In addition it also removes a bit trailing whitespace, Let me know if I should revert that.

Signed-off-by: Max Rosin <git@hackrid.de>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ The easiest way to try this, is by running a cluster with the CockroachDB Operat
 
 ```bash
 kubectl scale cockroachdbs basic --replicas 5 # scaling up from 3
-kubectl scale cockroachdbs basic --replicas 3 # scaling down from 3 (decommission of pod 3 and 4)
+kubectl scale cockroachdbs basic --replicas 3 # scaling down from 5 (decommission of pod 3 and 4)
```
Comment (metalmatze) [inline README.md:118]: That's actually correct the way it is. In the first command we scale up to 5 and in this command we scale down from 5. :blush:

Comment (ekeih) [reply README.md:118]: Hm? That's why I changed the the command to `down from 5`.

Comment (metalmatze) [reply README.md:118]: true that. Nevermind! Good catch!

File: README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Deploy [CockroachDB](https://www.cockroachlabs.com/product/) on [Kubernetes](htt
 
 ## YAML with Jsonnet & CockroachDB Operator
 
-The main focus of this project is to generate all necessary Kubernetes files, like StatefulSets, Services, CronJobs, and more with [Jsonnet](https://jsonnet.org).  
+The main focus of this project is to generate all necessary Kubernetes files, like StatefulSets, Services, CronJobs, and more with [Jsonnet](https://jsonnet.org).
```
Comment (metalmatze) [inline README.md:26]: This is actually a new line in markdown. I don't care that much though, so merging.

**Pull request merged**